components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.ts

```diff
@@ -212,24 +212,24 @@ export class ProfileOverviewComponent implements OnInit, OnDestroy {
   }
 
   onFilesSelected(event) {
-    this.sendFiles(Array.from(event.target.files)).subscribe(
-      fileUploads => { this.fileUploads = fileUploads; },
-      null,
-      () => {
+    this.sendFiles(Array.from(event.target.files)).subscribe({
+      next: fileUploads => { this.fileUploads = fileUploads; },
+      error: null,
+      complete: () => {
```

Comment:
Deprecated variant of subscribe:
```
subscribe is deprecated: Use an observer instead of a complete callback
```
That message is misleading; `subscribe` is _not_ wholly deprecated. The developers thought there were too many signatures for its own good (see https://github.com/ReactiveX/rxjs/issues/4159), so instead of using this (with three arguments)...
```
subscribe(next, err, complete) 
```
one needs to pass in a single argument (an "observer") like this:
```
subscribe({next: f1, err: f2, complete: f3})
```
That format also happens to make it simpler if you do not need to supply all three callbacks.

Reply:
🤔 So, I think you mean `error` in the last code block. But from what you say, it's not required to supply something that's not need -- so, I wonder why we'd put `error: null` into line 217.

Reply:
Yep, you're correct; that was just mechanical on my part, but don't need the `error: null`.